framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3239,6 +3239,10 @@ public AnnotatedWildcardType getUninferredWildcardType(AnnotatedTypeVariable typ
      * @param wildcard AnnotatedWildcardType whose upper bound is used to widen
      * @return {@code annotatedTypeMirror} widen to the upper bound of {@code wildcard}
      */
+    // TODO: BoundsInitializer#initializeExtendsBound(AnnotatedWildcardType) and
+    // SupertypeFinder#fixWildcardBound} have simalar logic for handling unbounded wildcards.
```

Comment:
The closing curly here should be `()` or removed? Same for the other two copies.